src/types.ts

```diff
@@ -26,14 +26,14 @@ export interface UpdateRTDBConfig {
 
 export interface TokensInfoCFConfig {
   gcloudProject: ValoraGcloudProject
-  environment: Environment
+  networkIds: NetworkId[]
 }
 
 export interface TokenInfo {
   name: string
   symbol: string
   decimals: number
-  tokenId: string
+  network: Network
```

Comment:
@jeanregisser the reason I didn't add network Id is because the tokens are grouped by network ID in the response, so consumers have this information already. I agree it could be handy in various situations, though

Reply:
I'm going to merge this for now since we have changes in progress that rely on it. Happy to discuss offline whether to add the network ID to TokenInfo as a followup

Reply:
ok I just realized what I said doesn't make sense. tokens are not grouped by network or networkID. sorry, Monday brain.

Reply:
I'll get a fix out in a sec